fastlane/Fastfile

```diff
@@ -35,27 +35,26 @@ my_android_app = ENV["APPCENTER_APP_NAME_ANDROID"]
 
 platform :ios do
   before_all do |lane, options|
-      env = assert_environment(**options)
+    env = assert_environment(**options)
   end
 
   desc "Build and upload to TestFlight"
   lane :upload do |options|
-    projectName = ENV["PROJECT_NAME"]
-    xcodeproj = "./ios/#{projectName}.xcodeproj"
-
-    scheme = case env
-    when "development"
-      "#{projectName}Dev"
-    when "staging"
-      "#{projectName}Stg"
-    when "production"
-      projectName
-    else
-      raise "Invalid environment: #{env}"
+    appName = ENV["APP_NAME"]
+    # my app name is HMezher Healthcare Demo
+    # so i need to change the app name to HMezherHealthcareDemo
+    appName = appName.gsub(" ", "")
+    workspace ="./ios/#{appName}.xcworkspace"
+    xcodeproj = "./ios/#{appName}.xcodeproj"
+
+    # Change to project root directory before running expo prebuild
+    Dir.chdir("..") do
+      sh "npx expo prebuild --clean"
```

Comment:
move npx expo repbuild outside fastlane. it should be a line of script above run fastlane

Reply:
okay, i got it